linux/devdoc/file_linux_requirements.md

```diff
@@ -0,0 +1,159 @@
+# file_linux

```

Comment:
>linux [](start = 7, length = 5)

Linux has these creepy errors with creepy names: EBADF is one of them.

Is there some support in this file (?) maybe logging? repo to have int corresponding to those errors printed as EBADF instead of (-3) or whatever it is?

Reply:
Added task

---
In reply to: [450474887](https://github.com/Azure/azure-c-pal/pull/7#discussion_r450474887) [](ancestors = 450474887)